lib/double_entry/line.rb

```diff
@@ -57,6 +57,8 @@ module DoubleEntry
   class Line < ActiveRecord::Base
     belongs_to :detail, polymorphic: true, required: false
     has_many :metadata, class_name: 'DoubleEntry::LineMetadata' unless -> { DoubleEntry.config.json_metadata }
+    scope :credits, -> { where('amount > 0') }
```

Comment:
This could be an embarrassingly dumb question. But are these scopes the wrong way round?

It seems to not match the implementation here: https://github.com/envato/double_entry/blob/dfdf61d8e692c9de0722395fa6416f32dd2b9e1e/lib/double_entry/transfer.rb#L107

Where a credit line will be created with a negative amount, and a debit line with a positive amount. But the scopes here do the inverse.

Reply:
No, you're absolutely right. It's even [in the specs](https://github.com/envato/double_entry/blob/dfdf61d8e692c9de0722395fa6416f32dd2b9e1e/spec/double_entry_spec.rb#L188). I honestly couldn't tell you what I was thinking when I wrote this ... oops!